_maps/map_files/Blueshift/BlueShift.dmm

```diff
@@ -142086,9 +142086,7 @@
 	dir = 1
 	},
 /obj/effect/turf_decal/stripes,
-/obj/machinery/recycler{
-	safety_mode = 1
```

Comment:
Recyclers are still valid for safety_mode

Reply:
Oh, I know, but I removed it because they're not meant to have that to 1 by default.